ui/apps/everest/src/pages/db-cluster-details/backups/backups-list/table-header/backups-list-table-header.module.css

```diff
@@ -0,0 +1,13 @@
+.scheduleToggleButton {
```

Comment:
I see that this selector is no longer used and there is instead inline styling for this component

Reply:
yep, good one!